tools/api-markdown-documenter/src/api-item-transforms/ApiItemTransformUtilities.ts

```diff
@@ -153,7 +153,9 @@ function getApiItemPath(
 
 	return [
 		fileName,
-		...documentAncestry.map((hierarchyItem) => config.getFileNameForItem(hierarchyItem)),
+		...documentAncestry.map((hierarchyItem) =>
+			getDocumentNameForApiItem(hierarchyItem, config),
```

Comment:
This is the entirety of the fix. The code wasn't calling the helper function that is used elsewhere to craft the document name.